dags/glam.py

```diff
@@ -209,18 +261,26 @@
         arguments=('--append_table','--noreplace',),
         dag=dag)
     clients_histogram_aggregates_merged_0 >> clients_histogram_aggregates_merged
-    clients_histogram_aggregates_merged >> clients_histogram_aggregates
+    clients_histogram_aggregates_merged >> clients_histogram_aggregates_0
 
-clients_histogram_bucket_counts = bigquery_etl_query(
-    task_id="clients_histogram_bucket_counts",
-    destination_table="clients_histogram_bucket_counts_v1",
-    dataset_id=dataset_id,
-    project_id="moz-fx-data-shared-prod",
-    owner="msamuel@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "msamuel@mozilla.com"],
-    date_partition_parameter=None,
-    arguments=('--replace',),
-    dag=dag)
+    clients_histogram_aggregates = bigquery_etl_query(
+        task_id="clients_histogram_aggregates_{}".format(partition),
+        destination_table="clients_histogram_aggregates_v1",
+        dataset_id=dataset_id,
+        project_id="moz-fx-data-shared-prod",
+        owner="msamuel@mozilla.com",
+        email=["telemetry-alerts@mozilla.com", "msamuel@mozilla.com"],
+        date_partition_parameter=None,
+        parameters=(
+            "min_sample_id:INT64:{}".format(min_param),
+            "max_sample_id:INT64:{}".format(max_param),
+        ),
+        arguments=('--append_table','--noreplace',),
+        dag=dag)
+    clients_histogram_aggregates_0 >> clients_histogram_aggregates
+    clients_histogram_aggregates >> clients_histogram_bucket_counts
```

Comment:
These two downstream tasks are causing the DAG to look complicated. A [`DummyOperator`](https://airflow.apache.org/docs/stable/_api/airflow/operators/dummy_operator/index.html?highlight=dummyoperator#module-airflow.operators.dummy_operator) may help. All the dependencies could point to this one task, then the dependencies rely on that task.

Reply:
This is mitigated a bit with the sub dags